Client/Frontend/Share/ShareExtensionHelper.swift

```diff
@@ -9,6 +9,24 @@ import OnePasswordExtension
 private let log = Logger.browserLogger
 
 class ShareExtensionHelper: NSObject {
+    
+    enum ShareActivityType {
+        case password
+        case iBooks
+        case `default`
+        
+        var description: String? {
```

Comment:
What is this `description` used for?

Reply:
I was using it for some debuggin but it is not needed, removing.